server/db/User.js

```diff
@@ -14,30 +14,46 @@ class User extends Model {
       firstName: {
         type: STRING,
         allowNull: false,
-        validate: {
-          len: [2, 100]
-        }
+        validate: { len: [2, 100] }
       },
       lastName: {
         type: STRING,
         allowNull: false,
-        validate: {
-          len: [2, 100]
-        }
+        validate: { len: [2, 100] }
       },
-      office: {
+      password: {
         type: STRING,
         allowNull: false,
-        validate: {
-          is: office
+        validate: { notEmpty: true },
+        private: true
+      }
+    };
+    const hooks = {
+      beforeCreate(user) {
+        return password.hash(user.password)
+          .then(hash => user.password = hash);
+      },
+      beforeUpdate(user) {
+        if (user.changed('password')) {
+          return password.hash(user.password)
+            .then(hash => user.password = hash);
         }
       }
     };
-    return super.init(fields, { sequelize });
+    return super.init(fields, { sequelize, hooks });
   }
 
   static associate(models) {
     User.hasMany(models.Liability);
+    User.belongsTo(models.Office);
+  }
+
+  // todo rewrite (check all keys, omit private ones)
```

Comment:
:+1: but be tidy:
```js
// TODO: here goes my todo
```
cause editors are able to list all todos across project if you respect that form.

Reply:
You should be aware that this also works (editor lists it), but anyway yours definitely looks nicer 💛

Reply:
There are more and less strict editors 😜

Reply:
> There are more and less strict editors 😜

Also can be loosely translated to: 'There are more or less crappy editors' 😛